### ISSUE TITLE
I can't generate the report 

### ISSUE THREAD

**Issue by Kcontreras** [uttesh/pdfngreport#29]
Hi Uttesh,
Firstly, thank you for your post it has been very helpful for me. I have tried to run a test case with the pdfngreport library, however after I did all settings, in the different files (pom.xml, pdfngreport.properties) the test case runs correctly, but the report is not generated, even the console does not shows any information or error regarding to the report. I am using jfreechart 1.0.13, fop 2.0, pdfngreport 2.1.3 for me project. I also have run your example on my machine (pdfngreportdemo-master) and this running successful but with my project doesn't work.  How can I know that over my project the library is working correctly? , Do I have some logs or something similar where I can see that everything is configure in the correct way? 

Thank for your attention and I hope you can help me with this.

**Comment by uttesh** [comment]
Hi Kcontreras,

  Could you please send me the build log i mean, when maven test command is
executed is there any error/warning? you can run the maven in the debug
mode and send that log.

Configuration looks fine to me, no issues is there.

maven command to get the all stack traces.

mvn test -e --log-file log.txt

send me that log.txt.

Thanks,
Uttesh

**Comment by Kcontreras** [comment]
Hi Uttesh, 
thank you for your quick response. I have just run the project on mode debug, so I attach you the log. I hope that this will be useful for you. Thank you very much for your care. 

Karen C.
[LogMvn.zip](https://github.com/uttesh/pdfngreport/files/1334081/LogMvn.zip)

**Comment by uttesh** [comment]
Hi Karen,

Gone through your log file, look like its having small mistakes in your pom
and JDK setting.

1. Its missing groupId in the pom.cml file. (line 112)

<plugin>

*<groupId>org.apache.maven.plugins</groupId>*
<artifactId>maven-compiler-plugin</artifactId>
<version>3.3</version>
<configuration>
<source>1.8</source>
<target>1.8</target>
</configuration>
</plugin>

2. make your using JDK not jre, check all setting is the code is compiling
properly. (logMvn.txt line 646)

Caused by: org.apache.maven.plugin.compiler.CompilationFailureException:
Compilation failure
No compiler is provided in this environment. Perhaps you are running on a
JRE rather than a JDK?

above error nothing do with the pdf report library.

Thanks,
Uttesh

**Comment by Kcontreras** [comment]
Yes Uttesh you are right, Let me check my errors and after that I am going to validate if it is possible generate the pdf report. 

Thanks,
Karen C.

**Comment by Kcontreras** [comment]
HI Uttesh, 
Firstly, thank you for your time, I have just corrected the errors and the debug running correctly and generated the build success. However, the report doesn't is generated and doesn't show errors regarding it. Another important thing is that the html report is generated successfully.

I attached the new log of mvn.

[LogMvn2.zip](https://github.com/uttesh/pdfngreport/files/1335577/LogMvn2.zip)

Thanks, 
Karen C.

**Comment by Kcontreras** [comment]
HI Uttesh,

Finally, I achieved to get the pdf report through  the pdf report library. However I have the last 
question, why does the log that is generated by testng is different that the report html?. I mean, the testng report that found a error then the test case fail, but the html report shows all successful. 

Once again thank you for your time,
Karen C.
[Reports.zip](https://github.com/uttesh/pdfngreport/files/1342555/Reports.zip)
